experiments/opencog/pattern_matcher_vqa/gui_helpers.py

```diff
@@ -0,0 +1,73 @@
+# setup pipeline
+import jpype
+from feature.image import ImageFeatureExtractor
+from ipywidgets import Image
+from scipy import misc
+from IPython.display import display
+from ipywidgets import Layout
+
+from ipywidgets import Button, Box, VBox, HBox
+
+from ipywidgets import widgets
+
+
+
+def build_extractor(prototxt, caffemodel):
```

Comment:
I propose using more verbose names, at first look I didn't understand what ```extractor``` is. For instance 'new_image_feature_extractor", "new_natural_language_question_converter", "initialize_atomspace_by_predefined_facts"